net-ftp/lftp/lftp-4.6.5.ebuild

```diff
@@ -32,9 +30,10 @@ RDEPEND="
 		>=net-proxy/dante-1.1.12
 		virtual/pam
 	)
+	gnutls? ( >=net-libs/gnutls-1.2.3 )
 	ssl? (
-		gnutls? ( >=net-libs/gnutls-1.2.3 )
-		openssl? ( dev-libs/openssl:0 )
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
 	)
```

Comment:
Yeah, I realize I'm changing the meaning of the USE flags. I figured the changes I made would make the usage of flags in lftp match the way we use the ssl and gnutls flags in the rest of the tree.

If those changes aren't a good idea, I can change it back.

I pulled the ebuild off of the bug report, which I submitted as a pull request to the LibreSSL project.